### PR TITLE
Add what we have removed section

### PR DESCRIPTION
**Pull request by satyamagarwal**

### CODE REVIEW

**Comment by bringdev** [comment]
[trello.com/c/PILjJ5cW](https://trello.com/c/PILjJ5cW)

File: api/shipping-guide_2/upgrade-to-version-2.html [hunk 1]
```diff
@@ -63,25 +63,103 @@ <h4>Introduced warnings</h4>
...
+            These tags are no longer available. From schema version 9, we introduced Consignments tag which consists of
+            all of these tags.
+          </p>
+
+          <p>
+            In shipping guide, every request can contain :
+          </p>
+          <ul>
+            <li>
+              One consignment with shipments details, product ids, and package details (packages can be one or many).
```
Comment (jonasac) [inline api/shipping-guide_2/upgrade-to-version-2.html]: Maybe change the wording to 'one or more consignment'

File: api/shipping-guide_2/upgrade-to-version-2.html [hunk 2]
```diff
@@ -63,25 +63,103 @@ <h4>Introduced warnings</h4>
...
+          <h3>What we have removed in Response</h3>
+          <h4>Trace messages</h4>
+          <p>
+            It is strongly recommended that trace messages should not be part of any integration.
+          </p>
+          <p>
+            This started out for stating an indication what's wrong with the request, but later the usage got extended.
           </p>
           <p>
+            We are keeping this for now, because still many users depend on it, but we will phase it out quickly.
```
Comment (jonasac) [inline api/shipping-guide_2/upgrade-to-version-2.html]: Maybe don't say anything about trace messages here

File: api/shipping-guide_2/upgrade-to-version-2.html [hunk 3]
```diff
@@ -63,25 +63,103 @@ <h4>Introduced warnings</h4>
...
           </p>
         </section>
         <section class="api__section">
           <h2>JSON interface</h2>
           <h3>What we have removed</h3>
-          <h4>Price adjustment</h4>
+          <h4>Fields with null values</h4>
           <p>
-            TODO:
+            In both JSON and XML REST response, you will not get fields with null values.
```
Comment (jonasac) [inline api/shipping-guide_2/upgrade-to-version-2.html]: No need to mention this

File: api/shipping-guide_2/upgrade-to-version-2.html [hunk 4]
```diff
@@ -63,25 +63,103 @@ <h4>Introduced warnings</h4>
...
-          <h4>XML is not supported (yet?)</h4>
           <p>
-            TODO: We do not support XML format for this interface anymore. Maybe we'll do that in the future, but right now....blablabla
+            The overall response of all the three REST APIs is same, and conforms Shipping Guide 2.0 schema.
+            So, response tags of REST API follow the previous section as well.
+          </p>
+
+          <h4>Price adjustment</h4>
+          <p>
+            TODO: TODO TODO TODO TODO
```
Comment (jonasac) [inline api/shipping-guide_2/upgrade-to-version-2.html]: This has been removed no ?

**Pull request merged**